packages/semcom-node/lib/main.ts

```diff
@@ -27,7 +28,12 @@ export const launch: (variables: Record<string, any>) => Promise<void> = async (
   await manager.configRegistry.register(configPath);
 
   const server: NodeHttpServer = await manager.instantiate('urn:semcom-node:default:NodeHttpServer', { variables });
+  const peerSyncScheduler: Scheduler  = await manager.instantiate('urn:semcom-node:default:PeerSyncScheduler', { variables });
+  const storageSyncScheduler: Scheduler  = await manager.instantiate('urn:semcom-node:default:StorageSyncScheduler', { variables });
+
   server.start();
+  peerSyncScheduler.start();
+  storageSyncScheduler.start();
```

Comment:
Hmm... does this work? I.e. are the schedulers the same objects used for the init of the server, not new ones?

Reply:
when I tested it, the server didn't instantiate the 2 schedulers at all, because they aren't needed I think? The only thing they need to share is memoryStorage and apparently that happens because it synchronizes when I launch 2 nodes